packages/zoe/src/zoeService/types.js

```diff
@@ -182,6 +182,7 @@
  * @property {() => Promise<OR>} getOfferResult
  * @property {() => void=} tryExit
  * @property {() => Promise<boolean>} hasExited
+ * @property {() => Promise<0|1>} wasWantSatisfied
```

Comment:
The name `wasWantSatisfied` definitely suggests a yes or no answer.
What name would work better with returning a count?

Reply:
`numWantsSatisfied` or simply `wantsSatisfied` since singular is by far more common than multi

Reply:
I like @turadg's suggestion of `numWantsSatisfied`.  (It beats out 'wantsSatisfied' because the latter can be read as boolean.

Reply:
> because the latter can be read as boolean.

fwiw, that's what I proposed as its virtue. when the caller doesn't know or care about multiples they don't have to. If they try to do anything with it that works for a boolean and not `0 | 1` the type checker will inform them.